openmeetings-webservice/src/main/java/org/apache/openmeetings/webservice/UserWebService.java

```diff
@@ -180,7 +180,7 @@ public UserDTO add(
 				user.setLanguageId(1L);
 			}
 			User jsonUser = user.get(userDao, groupDao);
-			IValidator<String> passValidator = new StrongPasswordValidator(true, jsonUser);
+			IValidator<String> passValidator = new StrongPasswordValidator(false, jsonUser);
```

Comment:
This validator needs to be false, otherwise the response will not put the actual error message into the output.